### ISSUE TITLE
Is Skeleton inactive?

### ISSUE THREAD

**Issue by davidnormo** [dhg/Skeleton#153]
Is Skeleton inactive?
I tried Skeleton out the other day and it was so nice and simple to use that I'd like to contribute. @dhg can you confirm the project is inactive now?

What's the future for this project if any? Have any of you started contributing to similar projects that you can point me towards?

**Comment by simonclay** [comment]
I am working up a version that responds to screen sizes 1024, 1280, 1440. But I have no idea how to share on GitHub. Any quick help would be appreciated.

I really like Skeleton because it's fixed widths mean that nested columns stay at the correct width unlike with percentage frameworks.

**Comment by davidnormo** [comment]
You will need to create a pull request - https://help.github.com/articles/creating-a-pull-request

That sounds useful but I wouldn't waste your time if Skeleton isn't being actively developed, your pull request would never get merged.

**Comment by catalinmiron** [comment]
@davidnormo you're right

@simonclay what you can do, is to fork skeleton and merge pull-request on your repo / or find an **active** repo for skeleton

**Comment by Mause** [comment]
A somewhat active fork by @nickpack;

https://github.com/nickpack/Ribs

**Comment by sroe** [comment]
dhg and 3 other developers are working on 2.0 as you can see in the 2.0-wip branch. So there will be a new version in the future I believe.

**Comment by Ansego** [comment]
Thats great news, if you guys require any help let me know.

Re

Ansego.

**Comment by dhg** [comment]
Hey all - this is my bad. I left Skeleton in awful disrepair and wasn't maintaining it at all for the past couple years. I realized that the project was still getting use and was something I wanted to continue pushing on, so recently released v2.0. I wrote a bit of a weird letter about the story here: https://medium.com/@dhg/dear-skeleton-452f4bb07d69

V2: http://www.getskeleton.com

I'm committed to maintaining Skeleton this time around and pushing it going forward, but I owe everyone in the community an apology for abandoning the project with no real status left on its future.
